presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -148,15 +148,15 @@ private PlanNodeStatsSummarizer() {}
             stats.add(new PlanNodeStats(
                     planNodeId,
                     new Duration(planNodeWallMillis.get(planNodeId), MILLISECONDS),
-                    planNodeInputPositions.get(planNodeId),
-                    succinctDataSize(planNodeInputBytes.get(planNodeId), BYTE),
+                    planNodeInputPositions.getOrDefault(planNodeId, 0L),
```

Comment:
Together with @szymonm we did some short investigation. It looks as it is related with the fact that index join source is processed by separate driver, and so stats collected by that driver are collected by main driver. 

Also I am not sure that index join source driver stats are merge-able with stats of the main driver.